### DISCUSSION TITLE
Get the current user in lookup

### DISCUSSION THREAD

**Discussion by belalAhmed** [serenity-is/Serenity#6404] [Help]
is anyone know how i can get the current user in lookup script class

![image](https://user-images.githubusercontent.com/3522191/170304998-b66983cf-12de-4167-b157-6e87f51b21de.png)

**Comment by minhhungit** [comment]
.NET CORE (you can check here **\Modules\Administration\User\Authentication\PermissionService.cs** )
```csharp
var userId = Convert.ToInt32(UserAccessor.User.GetIdentifier());
```

.NET FRAMEWORK:
```csharp
var userId = int.Parse(Serenity.Authorization.UserId)
```

**Reply by wezmag** [reply]
You might also want to check out the [multi tenancy lookup script tutorial](https://serenity.is/docs/tutorials/multi_tenancy/handling_lookup_scripts) to prevent cache issue.

**Reply by belalAhmed** [reply]
it's not work multi tenancy lookup  in .net core . Do you have any example

**Reply by belalAhmed** [reply]
> ```cs
> ```cs
> var userId = Convert.ToInt32(UserAccessor.User.GetIdentifier());
> ```
> 
> 
>     
>       
>     
> 
>       
>     
> 
>     
>   
> ```

not working in looup

**Reply by wezmag** [reply]
What was the error message?
